java/java.source.base/src/org/netbeans/api/java/source/SourceUtils.java

```diff
@@ -1085,22 +1090,15 @@ private static boolean mainCandidate(ExecutableElement method) {
         return true;
     }
 
+    // 0 is highest
```

Comment:
Thanks for adding this comment!  I just spent a couple of minutes looking at your loop wondering why the check was that way around. :laughing: